pages/results/challenge/[id].tsx

```diff
@@ -28,6 +28,10 @@ const ChallengeResultsPage: PageType = () => {
   const fetchGames = async () => {
     const res = await mailman(`scores/challenges/${challengeId}`)
 
+    if (res.notPlayed) {
+      return setNotAuthorized(true)
+    }
+
     if (res.error) {
       if (res.error.code === 401) {
```

Comment:
This check is no longer needed

Reply:
The `res.error.code === 401` check i mean

Reply:
Removed it